nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -353,6 +354,7 @@ sub in {
     my $fsType = $fields[$n];
     my $device = $fields[$n + 1];
     my @superOptions = split /,/, $fields[$n + 2];
+    $device =~ s/\\040/ /g; # account for devices with spaces in the name (\040 is the escape character)
```

Comment:
Are you sure device names can also have these special characters? At least the `fstab` manpage only mentions it for mount points.

Reply:
Yes.

mount | grep -i videos
data/Home Videos on /srv/Home Videos type zfs (rw,relatime,xattr,noacl)

Reply:
And both are escaped by `\040`?

Reply:
Alright, comment when you're done since github won't send email notifications on code changes.